### PR TITLE
The apply button is added to the editor

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
#### Description:
You can save an email in the editor. 

A little bit of information:
1. If the required field "Contact segment" is empty, the apply button won't be active.
2. If the required fields "Subject" and "Internal Name" are empty, the default value will be generated (e.g. Email 2022-01-01 12:01:00).
3. If the email form is not valid, the email will not be saved, and the modal will be displayed.

### CODE REVIEW

**Comment by volha-pivavarchyk** [comment]
How can I add a translation?
image

File: src/buttons/buttonApply.command.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  static getEmailType(emailForm) {
+    return emailForm.find('#emailform_emailType');
+  }
+
+  static getDefaultEmailName() {
+    return `E-Mail ${moment().format('YYYY-MM-D hh:mm:ss')}`;
+  }
+
+  static strcmp(str1, str2) {
```
Comment (adiux) [inline src/buttons/buttonApply.command.js]: @volha-pivavarchyk please add a description here too. so we know what this function is doing.

**Review by adiux — CHANGES_REQUESTED** [review]

**Comment by adiux** [comment]
@volha-pivavarchyk the same way you added the other text? If I remember correctly then you have to do it in the plugin.

**Comment by volha-pivavarchyk** [comment]
I removed some unused code

**Review by adiux — APPROVED** [review]

**Pull request merged**